client/apollo/css/src/Toggle/ToggleCommon.scss

```
@@ -11,13 +13,13 @@
     background-color: var(--toggle-bg-color);
   }
 
-  &:has(:focus-within, :focus-visible, :focus, :active) &__root {
+  &:has(:focus-visible) &__root {
     outline: 2px solid var(--toggle-focus-outline-color);
     outline-offset: 2px;
   }
 
-  &:has(:hover, :active) {
-    cursor: pointer;
+  &:has(:disabled) {
```

Cela devrait suffir je pense
```suggestion
  &:disabled {
```

---

l'état `disabled` n'est pas présent sur le &, mais plutôt sur un input descendant